src/cider/exec/nextgen/Nextgen.cpp

```diff
@@ -27,7 +27,7 @@
 
 namespace cider::exec::nextgen {
 
-std::unique_ptr<context::CodegenContext> compile(const RelAlgExecutionUnit& ra_exe_unit,
+std::unique_ptr<context::CodegenContext> compile(RelAlgExecutionUnit& ra_exe_unit,
```

Comment:
Why remove `const`? do we need to change ra_exe_unit during codegen?

Reply:
I used RelAlgExecutionUnit as a member variable in class InputAnalyzer, and traverse access to read and write members of it